csep/utils/plots.py

```diff
@@ -1232,29 +1232,84 @@ def _get_marker_style(obs_stat, p, one_sided_lower):
             fmt = 'gs'
     return fmt
 
-def plot_comparison_test(results, plot_args=None):
-    """Plots list of T-Test or W-Test Results"""
+def _get_marker_t_color(distribution):
+    """Returns matplotlib marker style as fmt string"""
+    if distribution[0] > 0. and distribution[1] > 0.:
+        fmt = 'green'
+    elif distribution[0] < 0. and distribution[1] < 0.:
+        fmt = 'red'
+    else:
+        fmt = 'grey'
+
+    return fmt
+
+def _get_marker_w_color(distribution):
+    """Returns matplotlib marker style as fmt string"""
+
+    if distribution < 0.05:
+        fmt = True
+    else:
+        fmt = False
+
+    return fmt
+
+def plot_comparison_test(results_t, results_w=None, plot_args=None):
+    """Plots list of T-Test (and W-Test) Results"""
+
     if plot_args is None:
         plot_args = {}
-    title = plot_args.get('title', 'CSEP1 Consistency Test')
+    title = plot_args.get('title', 'CSEP1 Comparison Test')
     xlabel = plot_args.get('xlabel', 'X')
     ylabel = plot_args.get('ylabel', 'Y')
+    ylims = plot_args.get('ylims', (None, None))
+    capsize = plot_args.get('capsize', 2)
+    linewidth = plot_args.get('linewidth', 1)
+    markersize = plot_args.get('markersize', 2)
 
     fig, ax = pyplot.subplots()
```

Comment:
can you replace this line with the following code? this will let us plot this using a previously initialized axes
```python
    if axes is None:
        fig, ax = pyplot.subplots(figsize=figsize)
    else:
        ax = axes
        fig = ax.get_figure()
```

Reply:
done, and added axes as  function argument

Reply:
awesome! ill merge this change once the CI is finished